sdk/search/search-documents/CHANGELOG.md

```diff
@@ -9,13 +9,17 @@
 ### Breaking Changes
 
 - Update `SearchIndexerDataNoneIdentity.odatatype`. [#23495](https://github.com/Azure/azure-sdk-for-js/pull/23495)
+- Fix `SearchClient.getDocument` with `selectedFields` option returning a wide result with `undefined` non-optional fields [#23627](https://github.com/Azure/azure-sdk-for-js/pull/23627)
 
 ### Bugs Fixed
 
+- Allow nested field names to be specified in `select` and `searchField` options [#23627](https://github.com/Azure/azure-sdk-for-js/pull/23627)
 - Fix discarded user-defined `onResponse` callback [#24479](https://github.com/Azure/azure-sdk-for-js/pull/24479)
 
 ### Other Changes
 
+- Add `object` type constraint to `IndexDocumentsClient` [#23627](https://github.com/Azure/azure-sdk-for-js/pull/23627)
```

Comment:
Seems like this was added to more than just this one client.

Reply:
It affects a number of types transitively, but I'll list them explicitly here.